dls-controls/malcolmjs
# Feature/fix link position #128

## Description

Fixed link positioning so it appears under the port.
Fixed links so the link can be seen when the block connects to itself.
Made the links splines.
Fixed some of the performance issues with the block details updating (work in progress)

## Testing instructions

Add a set up instructions describing how the reviewer should test the code
- Review code
- Check Travis build
- Review changes to test coverage
- Run `npm start` and navigate to http://localhost:3000/gui/PANDA/layout/PANDA:SEQ1
- Make links between blocks and link a block to itself.
- Use the React browser extension to highlight component updates, observe that the parent panel and the nav don't update now when the out LED on SEQ1 updates.

## Agile board tracking

connect to #128 
connect to #130 
connect to #145
